src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -103,25 +119,36 @@ function BaseHeader(props: HeaderProps) {
   const icons: { [key: string]: JSX.Element } = {
     backNav: getIcon(backAction || backActionDefault, <ArrowBackIcon />),
     edit: getIcon(navigateToEdit, <CreateIcon />, true),
-    user: getIcon(openProfileMenu, <AccountCircleIcon className={classes.account} fontSize="large" />),
+    user: getIcon(openProfileMenu, <MenuIcon className={classes.account} fontSize="large" />),
   };
 
   const left = leftIcon ? icons[leftIcon] : null;
   const header = title ? (
-    <Typography className={classes.title} variant="h2">
+    <Typography className={searchAction ? classes.leftTitle : classes.title} variant="h2">
       {title}
     </Typography>
   ) : null;
   const right = rightIcon ? icons[rightIcon] : null;
 
+  const onSearchExit = () => {
+    setSearchVisibility(false);
+    searchExit();
+  }
+
   return (
-    <div className={classes.root}>
+    <div className={classes.root} style={{ marginTop: props.searchAction ? '20px' : undefined }}>
```

Comment:
Don't introduce inline styles when you already have a class dedicated to this. You can do this conditionally based on props if you switch to `makeStyles` (see how this is done `OutlinedCardList` for an example).